source/extensions/filters/http/jwt_authn/BUILD

```diff
@@ -71,3 +69,51 @@ envoy_cc_library(
         "//source/extensions/filters/http/common:factory_base_lib",
     ],
 )
+
+envoy_cc_library(
+    name = "matchers_lib",
+    srcs = [
+        "matcher.cc",
+    ],
+    hdrs = [
+        "matcher.h",
+    ],
```

Comment:
if single file, just write it in single line as line 90

Reply:
done